### PR TITLE
Add release docs

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Addresses part of #219.

---

**Checklist:**

- [ ] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0 by @)")
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Release
+
+## Instructions for Redeploying Try PureScript
+
+After making a new compiler release, do the following to redeploy Pursuit using the new compiler.
```
Comment (thomashoneyman) [inline RELEASE.md:5]:
```suggestion
After making a new compiler release, do the following to redeploy Try PureScript using the new compiler.
```

Comment (thomashoneyman) [reply RELEASE.md:5]: Also, if there are breaking changes then we need to update the package set as described in the main README file.

Comment (JordanMartinez) [reply RELEASE.md]: Fixed the name change. I've also ported the package set docs and cleaned them up a bit. We're due for an upgrade as its still on a 0.14.3 package set.

**Review by thomashoneyman — APPROVED** [review]

**Comment by thomashoneyman** [comment]
@JordanMartinez want to take a stab at updating the packages and deploying again?

**Comment by JordanMartinez** [comment]
> want to take a stab at updating the packages and deploying again?

@thomashoneyman Sure. Any idea how we measure the package set size? Or is this just compiling the package set and seeing how large `output` is?

**Comment by thomashoneyman** [comment]
We’re only using less than 20% of the available memory on the server at peak, and I’ve been watching the package sets and it hasn’t significantly grown, and for that reason we can just go ahead and update it without worrying about it overwhelming the server

**Comment by JordanMartinez** [comment]
See #259.

**Pull request merged**